src/connectors/zeno.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+function isMainPlayer() {
+	return document.querySelector('.player-sticky') || null;
+}
+
+function isEmbedPlayer() {
+	return document.querySelector('#audioPlayer') || null;
+}
+
+// TODO add podcasts
+
+function setupMainPlayer() {
+	Connector.playerSelector = '.radio-main';
+	Connector.playButtonSelector = '.radio-audio-control [\\[hidden\\]="radioPlayer.playing"]';
+	Connector.trackSelector = '.radio-main .radio-song';
+	Connector.artistSelector = '.radio-main .radio-artist';
+}
+
+function setupEmbedPlayer() {
+	Connector.playerSelector = '.dashboard';
+	Connector.isPlaying = () => !document.querySelector('audio').paused;
+	Connector.trackSelector = '.dashboard-box__playing-song';
+	Connector.artistSelector = '.dashboard-box__playing-artist';
+}
+
+(function() {
+	if (isMainPlayer()) {
+		setupMainPlayer();
+		return;
+	}
+	if (isEmbedPlayer()) {
+		setupEmbedPlayer();
+		return;
```

Comment:
Is there anything we can do about this to keep the lint from worrying about these same constructs? Such a twin blocks will make it easy to swap blocks and add new ones in the future.

Reply:
Since these are exclusive, they can be trivially converted to `else if`, or am I wrong?